util/scripts/create-symlink.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2022 Mischback
+# SPDX-License-Identifier: MIT
+# SPDX-FileType: SOURCE
+
+# Create a symbolic link to the actual config file.
+#
+# @param $1 The link's name
+# @param $2 The target of the link, the file to be referenced
+#
+# The script will create a backup of existing files / directories, see
+# ``util/scripts/backup-existing-target.sh``.
```

Comment:
- [ ] Verify, that this script is still in use (assumption: **no**)
- [ ] Decide if it should be kept (probably: **no**)